Source/Krypton Components/Directory.Build.targets

```diff
@@ -140,15 +140,10 @@
 
 
 				<Description Condition="'$(TFMs)' == 'all'">
-					An update to Component factory's krypton toolkit to support .NET Framework 4.6.2 - 4.8.1, .NET Core 3.1 and .NET 5 - 7. $(Description)
-
-					This package supports all .NET Framework versions starting .NET Framework 4.6.2 - 4.8.1, .NET Core 3.1 and .NET 5 - 7.
-					Also, all libraries are included targeting each specific framework version for performance purposes.
-
-					To view all of the standard toolkit package latest version information, please visit: https://github.com/Krypton-Suite/Krypton-Toolkit-Suite-Version-Dashboard/blob/main/Documents/Modules/Standard/Krypton-Toolkit-Suite-Standard-Modules.md
-
+					An update to Component factory's krypton toolkit to support .NET Framework 4.6.2 - 4.8.1, .NET Core 3.1 and .NET 6 - 7. $(Description)
```

Comment:
This big space at the beginning of each line can be removed ?

Reply:
![image](https://user-images.githubusercontent.com/949607/208652928-15a234e1-c852-49ba-a434-7a53d40c55a4.png)

Reply:
Nope, 
![image](https://user-images.githubusercontent.com/2418812/208653436-60484b38-f027-45d6-80bb-95ee42b09d54.png)

Every line needs to have the preamble spacing removed, otherwise it is displayed in the Nuget description in Visual Studio and looks weird and wastes real estate (See bug picture for VS display)

Reply:
Done, though don't open in VS, as it likes to word wrap!

Reply:
![image](https://user-images.githubusercontent.com/949607/208662204-cdc81220-46a3-4a2e-aa16-d948001849c5.png)